src/formattedcode/output_spdx.py

```diff
@@ -245,6 +245,9 @@ def write_spdx(
     ns_prefix = '_'.join(package_name.lower().split())
     comment = notice + f'\nSPDX License List: {scancode_config.spdx_license_list_version}'
 
+    version_major, version_minor = tuple(scancode_config.spdx_license_list_version.split("."))
```

Comment:
IMHO tuple is not needed there: "tuple unpacking" works from iterables
```suggestion
    version_major, version_minor = scancode_config.spdx_license_list_version.split(".")
```

Reply:
Fixed, thanks!